src/utils/vc/crypto/custom-linkeddatasignature.js

```diff
@@ -4,20 +4,68 @@ import base64url from 'base64url';
 
 const MULTIBASE_BASE58BTC_HEADER = 'z';
 
-function createJws({ encodedHeader, verifyData }) {
+const detachedHeaderParams = {
+  b64: false,
+  crit: ['b64'],
+};
+
+// Taken from https://github.com/transmute-industries/verifiable-data/blob/main/packages/jose-ld/src/JWS/createSigner.ts
```

Comment:
don't suppose they have a library we could just borrow instead of copying the code?

Reply:
unfortunately they dont export just this method in their library

Reply:
theres also slight modifications